modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -2,7 +2,9 @@
 
 This guide shows how to provision new Fedora CoreOS (FCOS) nodes on the VMware hypervisor.
 
-NOTE: Only https://kb.vmware.com/s/article/1003746[hardware version] 13 or later, vSphere ESXi hosts version 6.5 or later and vCenter host version 6.5 or later are supported.
+NOTE: Only https://kb.vmware.com/s/article/1003746[hardware version] 13 or later, vSphere ESXi hosts version 6.5 or later and vCenter host version 6.5 or later are supported. It may be possible to
+xref:modifying-ovf-metadta.adoc[modify the metadata of the OVA] to run in older versions of vSphere or vCenter,
```

Comment:
This mentions OVA, but the title of the other document we're adding mentions OVF.. As someone dumb to the vmwhare sphere, what's the difference?

Reply:
Ahh I see a paragraph in the new doc about it "shipped as an OVA, which is simply a tarball that contains the files disk.vmdk and coreos.ovf"

Reply:
in that case maybe we can say "modify the OVF metadata of the OVA" here.